worldedit-core/src/main/java/com/sk89q/worldedit/extension/platform/Platform.java

```diff
@@ -29,6 +30,7 @@
 import com.sk89q.worldedit.world.World;
 import com.sk89q.worldedit.world.registry.Registries;
 import org.enginehub.piston.CommandManager;
+import org.jetbrains.annotations.NotNull;
```

Comment:
Are we using javax or jetbrains for the most part?

Reply:
Jetbrains annotations seem to be used more frequently